src/style.css

```diff
@@ -1 +1,72 @@
-/* Styles go here */
+body {
+  margin: 0;
+}
+
+.header {
+  position: relative;
+  display: flex;
+}
+
+.nav {
+  display: flex;
+  align-items: center;
+  position: absolute;
+  right: 50px;
+  top: 50%;
+  transform: translateY(-50%);
+  height: 60px;
+}
+
+.logo {
+  margin: 10px 0;
+  height: 40px;
+  margin-left: 50px;
+}
+
+.nav__list {
+  display: flex;
+  align-items: center;
+  padding: 0;
+  margin: 0;
+  list-style: none;
+  height: 60px;
+}
+
+.nav__list li {
```

Comment:
don't use tags to implement styles! By doing this you are hang the specificity.

Check the cheklist